src/components/ExtendedReminder/index.tsx

```diff
@@ -129,23 +134,25 @@ const ExtendedReminder: FunctionComponent<Props> = ({
       <Typography mb={1} variant="bodyD">
         {text}
       </Typography>
-      {linkText && (
-        <StyledButtonLink
-          variant="link"
-          to={to}
-          onClick={onClick}
-          size="xsmall"
-          icon="east"
-        >
-          {linkText}
-        </StyledButtonLink>
-      )}
+      {CustomLink ||
+        (linkText && (
+          <StyledButtonLink
+            variant="link"
+            to={to}
+            onClick={onClick}
+            size="xsmall"
+            icon="east"
+          >
+            {linkText}
+          </StyledButtonLink>
+        ))}
```

Comment:
```suggestion
        (linkText ? (
          <StyledButtonLink
            variant="link"
            to={to}
            onClick={onClick}
            size="xsmall"
            icon="east"
          >
            {linkText}
          </StyledButtonLink>
        ) : null)}
```
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx/ ?

Reply:
@DorianMaliszewski `linkText` is undefined if not set, so in case its value is undefined nothing will render I believe

Reply:
As you want but I think its a good practices to use ternary instead of && to avoid some mistakes 🤔